BRANCHING.md

```diff
@@ -6,18 +6,23 @@ for code reviews. These branches are not the focus of this document, as they
 are generally short lived (as short as a few minutes, or as long as it takes
 for a developer to complete his work).
 
-When working on branch “develop” (equivalent of the improvement branch on Team
+When working on branch “master” (equivalent of the improvement branch on Team
 Foundation Server, our prior source code management system):
 
-1. We will “normally” have one development stream: Branch “develop”. This is
+1. We will normally have one development stream: Branch “master”. This is
 the equivalent of the “improvement” branch in TFS, and represents our
 “latest/greatest” code. Most developers work on this branch, or a sub-branch
 of this branch, the majority of the time.
 
-2. We will have a “master” branch; we will merge to this when we want to test
-and stabilize for a release (if needed). Note that the “master” branch will
-generally be the last “stable” release of the “develop” branch; no other
-branches should be merged there. Once stabilized, we'll tag it for release.
+2. When we need a branch to stabilize a release, we will create a branch on the
```

Comment:
do you mean "create a branch" on the super-project or the individual repository?